upload/admin/model/user/user.php

```diff
@@ -155,6 +155,46 @@ public function getUsers(array $data = []): array {
 	public function getTotalUsers(): int {
 		$query = $this->db->query("SELECT COUNT(*) AS `total` FROM `" . DB_PREFIX . "user`");
```

Comment:
You forgot to change this:
```suggestion
		$sql = "SELECT COUNT(*) AS `total` FROM `" . DB_PREFIX . "user`";
```